homeassistant/components/ephember/climate.py

```diff
@@ -180,6 +182,9 @@ def max_temp(self):
         if self._hot_water:
             return zone_target_temperature(self._zone)
 
+        if self._immersion:
+            return 90.0
+
         return 35.0
```

Comment:
Same comment about magic numbers.

Reply:
removed self._immersion. Needed to set an absolute max for the range for hot water